bigcommerce-labs/dev-docs
# DEV-22 - Swagger to slate markdown script for V3

#### What?

A script that will take raw Swagger files and convert them to slate-friendly markdown for easy including into the docs.

#### How To Test

Run `node swagger2md.js` from project root, with valid swagger .yaml files in source/api/v3

You should expect any .yaml.md files in the directory to be deleted and new generated files to be created in their place from the valid .yaml files.

Must `npm install` these first:
widdershins
js-yaml

The intent is that changes to the raw .yaml files can be committed, after which this script can be run and the updated .yaml.md files are then committed and deployed.

ping @bc-AlyssNoland @becomevocal
